arm-resources/locks/2015-01-01/swagger/locks.json

```diff
@@ -547,9 +588,6 @@
             "type": "string",
```

Comment:
Yes, this has been there for quite a while. We can try removing it and see how the generated code look like.